src/Features/Core/Portable/UseThrowExpression/UseThrowExpressionCodeFixProvider.cs

```diff
@@ -54,13 +55,25 @@ public override Task RegisterCodeFixesAsync(CodeFixContext context)
                 var throwStatementExpression = root.FindNode(diagnostic.AdditionalLocations[1].SourceSpan);
                 var assignmentValue = root.FindNode(diagnostic.AdditionalLocations[2].SourceSpan);
 
-                // First, remote the if-statement entirely.
+                var text = document.GetTextSynchronously(cancellationToken);
```

Comment:
don't get this synchronously.  you're in an async conext.  so you can get it with the normal GetTextAsync.